### ISSUE TITLE
integration with specimen micropublication

### ISSUE THREAD

**Issue by seltmann** [phenoscape/TraitFest-2023#15]
We are creating github repos and versioned zenodo pubications for sharing images and data about specimens from our project. There are many reasons to do micropublication based on specimens, including enabling linking to single specimens, providing all images for datasets used in photogrammetry, etc.

Here is an example:
https://github.com/Big-Bee-Network/UCSB-IZC00012194

How could we integrate trait information into these specimen micropublications to 1) increase findability and 2) provide trait information to be usable for wikidata or other derivative products?

**Comment by JCGiron** [comment]
"Traditionally" the way that annotations for anatomical structures have been incorporated into publications has been by adding an IRI/URI table (for example: https://dfzljdn9uc3pi.cloudfront.net/2016/2682/1/Table_S3.pdf from https://peerj.com/articles/2682/).

This issue can also be understood as how to annotate images, which would be related to https://github.com/phenoscape/TraitFest-2023/issues/7

**Comment by seltmann** [comment]
@JCGiron true! And that would be a good first addition for describing what structures are visible in the images. Specially we are recording body size measurements for 100/1000s specimens for the project. These are what I would like to include in the specimen publication. Perhaps with a methodology similar to https://pubmed.ncbi.nlm.nih.gov/23652347
